nomad/constants.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+
+
+# CONTAINERS
+# --
+
+CONTAINER_STOPPED = 102
```

Comment:
Well depending on the Python 3 versions we want to support, we could have used the built-in ``Enum``. ;-) But I don't think it is necessary to wrap this kind of values in classes for now. We could also put these values in the new ``Container`` class. What do you think?